modules/serverless-ossm-setup-without-kourier-mtls.adoc

```diff
@@ -0,0 +1,199 @@
+[id="serverless-ossm-setup-without-kourier-mtls_{context}"]
+= Integrating {ProductShortName} and {ServerlessProductName} with mTLS enabled
+
+.Procedure
+
+. Create a `ServiceMeshControlPlane` object in the `istio-system` namespace that has mTLS enabled.
```

Comment:
I don't follow you on this. What do you think I should report precisely?

Those are our docs, and I'm raising a concern that this single sentence is there to define a procedure step, that might not be understood well enough just by this sentence itself.

Maybe some SM experts would know what we mean by _"namespace that has mTLS enabled"_, but I think most users would need at least a confirmation. A link to longer description gives that confirmation/extra knowledge if needed. The same we do in our docs when we say: _"Install Serverless Operator"_ - this is always linked to a proper page, even on advanced topics.

Reply:
@cardil this is a procedure module - we can't add links to these yet anyway, only to assemblies. If you see "modules/" before something in a PR, we can't add links there yet. It's part of the OCP docs guidelines, but afaik they are planning to change it. For now, we can't do it though.

Re Kenjiro's comment, I'd agree, it's not clear from your comment whether you mean we need to add something to the service mesh docs, since both of the links you provided are to service mesh docs.

> I think this is unclear what to modify in SMCP as even SM configuration reference page, doesn't contain a "mTLS" text.
> This should be linked somewhere. Maybe here: https://deploy-preview-33179--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-security.html#ossm-security-enabling-strict-mtls_ossm-security

In both of those links in your comment ^^ if something needs to be added that's for mTLS and service mesh, it does need to be requested from the service mesh team.

Lastly, I'd say that if someone is configuring SM with Serverless and mTLS, it's not a huge leap to assume that they should know what "namespace that has mTLS enabled" should mean, and how to do this with Service Mesh prior to doing this integration. Otherwise you could argue that we should also explain how to create a `ServiceMeshControlPlane` object - we shouldn't. We assume this knowledge because it's not specific only to Serverless and should be adequately covered in the SM docs.